android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -56,6 +65,10 @@
 
 public class BuySendSwapActivity extends AsyncInitializationActivity
         implements ConnectionErrorHandler, AdapterView.OnItemSelectedListener {
+    private final static String TAG = "BuySendSwapActivity";
+    private final static String ETHEREUM_CONTRACT_FOR_SWAP =
+            "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee";
```

Comment:
Why is this an invalid address needed? I'm guessing we do the same on desktop so just asking for me.

Reply:
0x recommends using contract addresses for identifying assets, instead of symbols. I think it's because two assets can share the same ticker, but never the same contract address. `0xee...` is just how 0x identifies ETH.